pandas_gbq/gbq.py

```diff
@@ -1028,24 +1030,30 @@ def exists(self, table_id):
         from google.api_core.exceptions import NotFound
 
         table_ref = self.client.dataset(self.dataset_id).table(table_id)
+
         try:
-            self.client.get_table(table_ref)
+            table = self.client.get_table(table_ref)
+            if self.is_date_partitioned(table_id):
+                return table.num_rows > 0
+
             return True
         except NotFound:
             return False
         except self.http_error as ex:
             self.process_http_error(ex)
 
-    def create(self, table_id, schema):
+    def create(self, table_id, schema, date_partitioned=False):
```

Comment:
Rather than pass in `date_partitioned` here, I'd like to see an argument similar to the configuration argument in `read_gbq` which can take a dictionary in the JSON API format. That way options like `timePartitioning` can be set.

Reply:
That makes sense to me! I'll look into including the configuration in the `to_gbq` call to enable this and rest of the API configuration.